src/scripts/templates/gift_card.js

```diff
@@ -4,12 +4,10 @@
  * A file that contains scripts highly couple code to the Gift Card template.
  */
 
-import $ from 'jquery';
-
 const config = {
   qrCode: '#QrCode',
-  printButton: '#PrintGiftCard',
-  giftCardCode: '.giftcard__code',
+  printButton: 'PrintGiftCard',
```

Comment:
We've been slowly migrating our selectors to use data attributes in order to between seperate concerns between HTML, CSS, and JS. Lets do the same here since we're touching the code. Take a look at [`product.js`](https://github.com/Shopify/starter-theme/blob/master/src/scripts/sections/product.js) for what I mean.